lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -136,5 +140,15 @@ def _get_s6a_manager(service, processor):
     )
 
 
+def _randomize_sync_interval(interval: int) -> int:
+    """_randomize_sync_interval increases sync interval to ameliorate
+    thundering herd at Orc8r."""
```

Comment:
Please use google style docstring for the function, that will also fix the python linting comments